packages/configtest/package.json

```diff
@@ -3,7 +3,12 @@
   "version": "1.2.0",
   "description": "Validate a webpack configuration.",
   "type": "module",
-  "exports": "./lib/index.js",
+  "exports": {
+    "./package.json": "./package.json",
+    ".": {
+      "import": "./dist/index.js"
```

Comment:
Always add `default` as fallback :+1:

Reply:
Yes, webpack v5 is using [require](https://github.com/webpack/webpack/blob/9fcaa243573005d6fdece9a3f8d89a0e8b399613/bin/webpack.js#L69) as well.